Submitty/Submitty
# [Feature:TAGrading] TA Grading Touch Panel Resize

### Please check if the PR fulfills these requirements:


### What is the current behavior?
You cannot resize the TA grading panels when using only a touchscreen.

### What is the new behavior?
You can resize the TA grading panels with a touchscreen.

Tested on a Lenovo touchscreen running Brave 1.22.71 (Chromium 89.0.4389.114), and an iPad (iOS 14.2) - note: currently cannot scroll in panel selection popup on a touchscreen.

Steps for testing on a remote device:
1) Stop vagrant machine if running
2) In Vagrantfile, add a new line in config.vm.define under the site, websockets, and database ports - `ubuntu.vm.network 'forwarded_port', guest: 22, host: <port>, id: "ssh", host_ip: "0.0.0.0"`, replacing port with the port you want to expose for ssh.
3) Expose ports if necessary on device
4) Start vagrant machine with `vagrant up ssh` (or at the end of the Vagrantfile, comment out the if and end for ARGV.include?)
5) Retrieve SSH keys for vagrant machine from (`<SUBMITTY GIT REPO>/.vagrant/machines/<VM>/<VM BACKEND>/private_key`; on my machine the VM is ubuntu-18.04 and the VM backend is virtualbox)
6) Connect remote device to same network as local machine, use SSH utility such as Blink or Terminus (iOS apps, there are probably others for both iOS/Android/other devices) to connect to the port on the local machine with username root and private key from above; when you SSH, use parameter "-L" to forward site and websocket ports from local machine to remote device (ex. `ssh -L 1501:localhost:1501 -L 8443:localhost:8443 root@$SUBMITTY_HOST` , where $SUBMITTY_HOST is the local machine address)
     1) If you are on iOS, make sure that the SSH utility that you're using supports location tracking to keep the connection active in the background (for Blink, use `geo track`)
7) On remote device, navigate to localhost:1501
